definitions/private-law/json/AuthorisationCaseField/ManageOrders/AuthorisationCaseField.json

```diff
@@ -999,5 +1020,1357 @@
     "CaseFieldID": "orderCollection",
     "UserRole": "caseworker-privatelaw-systemupdate",
     "CRUD": "CRU"
+  },
+  {
+  "LiveFrom": "7/05/2022",
+  "CaseTypeID": "PRLAPPS",
+  "CaseFieldID": "courtName1",
+  "UserRole": "caseworker-privatelaw-courtadmin",
+  "CRUD": "CRU"
+},
+  {
+    "LiveFrom": "7/05/2022",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "courtName1",
+    "UserRole": "caseworker-privatelaw-solicitor",
```

Comment:
I think for manage order fields, we don't need solicitor permission. Can we please remove them from all the fields?

Reply:
Solicitor has read access so that should be fine